src/xSTIR/pSTIR/STIR.py

```diff
@@ -1581,15 +1607,29 @@ def domain_geometry(self):
 
     @property
     def subset_num(self):
+        '''selected subset number used for forward and back projection
+        
+        default value is 0 and corresponds to forward/backward projecting
+        the whole dataset.
```

Comment:
this is incorrect

Reply:
what is not correct?

Reply:
this is only true if `num_subsets` is 1, otherwise subsets other than the first are not projected

Reply:
I still don't really understand what is wrong, but I tried to rephrase it.

Reply:
if `num_subsets` is 2, and `subset_num` is 0, then subset 0 is projected and subset 1 is not

just drop what you wrote after "default value is 0"